mySqlDatabaseRemote/src/mySqlDriver2/InventoryDataBase.java

```diff
@@ -119,7 +119,165 @@ else if(menuSelection.equals("Delete an existing record")){
    }
    
    private void deleteRecord() {
-    }
+	           // Retrieve the content-pane of the top-level container JFrame
+      // All operations done on the content-pane
+
+      JTextField searchPid;  // Use Swing's JTextField instead of AWT's TextField
+      JButton submitButton;    // Using Swing's JButton instead of AWT's Button
+      
+      Container cp = getContentPane();
+      cp.setLayout(new FlowLayout());
+      //This panel will make so it will make a new row for the textFields
+      JPanel panelOne = new JPanel();
+      panelOne.setLayout(new FlowLayout());
+      panelOne.setVisible(true);
+
+      JLabel text = new JLabel("Enter the Product ID of the item to be deleted.");
+      panelOne.add(text);
+
+      JPanel panelTwo = new JPanel();
+      searchPid = new JTextField(10);
+      panelTwo.add(searchPid);
+
+      //Once the record is found the data will be displayed in this area
+      //This panel wont be visible until the search for the product id is made
+      JPanel panelThree = new JPanel();
+      panelThree.setVisible(false);
+      //This is what the data will be displayed on 
+      JTextPane textPane = new JTextPane();
+      textPane.setEditable(false);
+      panelThree.add(textPane);
+      //This is the confirmation that they want to delete a record, action is brought up later
+      JButton yes = new JButton("Yes");
+      yes.setForeground(Color.RED);
+      //Will take th user back to the main menu if they do not want to delete a record after all
+      JButton noButton = new JButton("No!");
+      noButton.setForeground(Color.green);
+      noButton.addActionListener(new ActionListener() {
+        @Override
+        public void actionPerformed(ActionEvent e) {
+           System.out.println("The record was NOT deleted");
+           cp.removeAll();
+           displayMenue();
+        }
+     });
+    
+      panelThree.add(noButton);
+      
+     //This button is used to make a query to MySQL if user enters a valid product id
+      submitButton = new JButton("Submit");
+      submitButton.addActionListener(new ActionListener() {
+        @Override
+        public void actionPerformed(ActionEvent e) {
+            int fieldselection = 0;
+            //Wont go make a search until there is a valid product id
+            if(checkInteger(searchPid.getText())){

```

Comment:
I didn't see this method did you mean to add this one too?

Reply:
That's just for the button, its not not really a method by itself